### PR TITLE
Update free-programming-books-subjects.md

### PR DESCRIPTION
**Pull request by vishalkrishna8**
## What does this PR do?
Add resource(s) 

### Description
Added a free book on OOP named Learn Object Oriented Thinking and Programming by Rudolf Pecinovsky. 

### Why is this valuable (or not)?
I found it incredibly informative and you can learn by reading yourself

### How do we know it's really free?
The book is provided in author, anyone can download from the website. 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
title page says "OOP – Learn
Object Oriented
Thinking and
Programming"

**Comment by vishalkrishna8** [comment]
> title page says "OOP – Learn Object Oriented Thinking and Programming"



> title page says "OOP – Learn Object Oriented Thinking and Programming"

That's just a title, this book explains in detail about the OOP and I believe that whoever reads this book once, most of his doubts will be cleared.
Thank you.

**Comment by eshellman** [comment]
Don't invent titles! Use the title given by the resource.

**Comment by vishalkrishna8** [comment]
> Don't invent titles! Use the title given by the resource.

Updated title as per given by the resource.

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -661,6 +661,10 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 ### Object Oriented Programming
 
 * [Object Oriented Programming](https://www.cl.cam.ac.uk/teaching/0910/OOProg/OOP.pdf) - Robert Harle (PDF)
+* [OOP – Learn
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:664]: needs to be on one line

Comment (vishalkrishna8) [reply books/free-programming-books-subjects.md]: I did as you wanted

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**